<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndClass</class>
 <widget class="QMainWindow" name="wndClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translate</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/icon.png</normaloff>:/res/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QTextEdit" name="textFrom"/>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="langFrom"/>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="swap">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/swap.png</normaloff>:/res/swap.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="langTo"/>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="translate">
      <property name="text">
       <string>Translate</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QTextEdit" name="textTo">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
